### PR TITLE
Beatmap descriptions

### PR DESCRIPTION
**Pull request by lexiolvsu**
I decided to make a stub about Beatmap descriptions, and I think I did a good job, but I think it could be more complete.
Tell me what you think! Edit it if you want.

### CODE REVIEW

File: wiki/Beatmaps/Beatmap_description/en.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+stub: true
+---
+
+## Beatmap description
```
Comment (paper-toaster) [inline wiki/Beatmaps/Beatmap_description/en.md:5]:
```suggestion
# Beatmap descriptions
```

File: wiki/Beatmaps/Beatmap_description/en.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+**Beatmap descriptions** are small areas under a [beatmap](/wiki/Beatmaps)'s page that give players information about said beatmap using [BBCode](/wiki/BBCode). Most [Ranked](/wiki/Beatmaps#ranked) beatmaps use at least one image, a number (depending on how many beatmaps the creator has Ranked), and a difficulty section.
+
+## Good beatmap descriptions
+
+*Notice: In order to get a beatmap ranked, it is recommended you have a good beatmap description, but this is not required.*
```
Comment (paper-toaster) [inline wiki/Beatmaps/Beatmap_description/en.md:11]:
```suggestion
*Notice: In order to get a beatmap ranked, it is recommended that creators have a good beatmap description, but this is not required.*
```

File: wiki/Beatmaps/Beatmap_description/en.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+---
+stub: true
+---
+
+## Beatmap description
+
+**Beatmap descriptions** are small areas under a [beatmap](/wiki/Beatmaps)'s page that give players information about said beatmap using [BBCode](/wiki/BBCode). Most [Ranked](/wiki/Beatmaps#ranked) beatmaps use at least one image, a number (depending on how many beatmaps the creator has Ranked), and a difficulty section.
+
+## Good beatmap descriptions
```
Comment (paper-toaster) [inline wiki/Beatmaps/Beatmap_description/en.md:9]:
```suggestion
## Uses and information
```

File: wiki/Beatmaps/Beatmap_description/en.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+*Notice: In order to get a beatmap ranked, it is recommended you have a good beatmap description, but this is not required.*
+
+A good beatmap description would include:
```
Comment (paper-toaster) [inline wiki/Beatmaps/Beatmap_description/en.md:13]:
```suggestion
Popular beatmap descriptions often include the following:
```

File: wiki/Beatmaps/Beatmap_description/en.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+- A list of all the guest difficulties and their creators
+- An "18+ lyrics" warning (if neccessary)
+- Information that would help a player decide whether they want to download the beatmap or not
+- An edits list or changelog
+
+### Images
+
+Many beatmap descriptions contain at least one image. To learn know how to add a image using BBCode, check out the [BBCode article](/wiki/BBCode#image). Also, make sure you have the link of the image you want to put in the beatmap description.
```
Comment (paper-toaster) [inline wiki/Beatmaps/Beatmap_description/en.md:22]:
```suggestion
Many beatmap descriptions contain at least one image (often an animated [GIF](https://en.wikipedia.org/wiki/GIF "Wikipedia") of some sort), that are used to express and illustrate either the creator, song, and/or beatmap in a flashly way. They can also take places of a title and/or header to add flair and quality. To learn how to add an image using BBCode, check out the [BBCode article's tutorial on images](/wiki/BBCode#image).
```
mostly just added more information here

File: wiki/Beatmaps/Beatmap_description/en.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+### Difficulty sections
+
+A difficulty section is located in a beatmap description, usually this section has a list of all the guest difficulties and their creators, this is mostly used to showcase the difficulties of a beatmap. Difficulty sections usually have images for the icons.
+
+#### Difficulty icons
+
+The difficulty icons can be found in the [Difficulties article](/wiki/Difficulties).
```
Comment (paper-toaster) [inline wiki/Beatmaps/Beatmap_description/en.md:30]:
```suggestion
### Difficulty sections

A "difficulty section" is a list of all the guest difficulties and their creators located in a beatmap description. This is mostly used to showcase the difficulties of a beatmap. Difficulty sections usually have images for the icons.

*Note: The difficulty icons can be found in the [Difficulties article](/wiki/Difficulties).*
```

**Review by MegaApplePi — APPROVED** [review]

**Pull request merged**